docs/deploy.md

```diff
@@ -134,9 +148,33 @@ For the official homeworld servers:
                 GSSAPIAuthentication yes
                 GSSAPIKeyExchange no
                 GSSAPIDelegateCredentials no
+                LocalForward 5901 localhost:5901
+                LocalForward 5902 localhost:5902
+                LocalForward 5903 localhost:5903
+                LocalForward 5904 localhost:5904
+                LocalForward 5905 localhost:5905
+                LocalForward 5906 localhost:5906
+                LocalForward 5910 localhost:5910
     $ scp preseeded.iso toast:/srv/preseeded.iso
+        # If you get a password prompt, DON'T give it anything.
+        # You should generate Kerberos tickets (kinit) before
+        # attempting to access the development server, otherwise
+        # SSH will fallback to password authentication.
+
+# Setting up the machines
+
+## For development only: Rebuilding the virtual machines
+
+For development, we're using a set of virtual machines on toast. To simulate cluster bringup, we destroy all the virtual machines and rebuild them using a script on toast. On toast (access toast with ``ssh toast``):
 
-## Set up the supervisor operating system
+    $ ~/hyades/rebuild-homeworld-cluster.sh /srv/preseeded.iso
+
+You can then access the virtual machines using VNC. For example, using TigerVNC:
+
+    $ vncviewer localhost:5910 # supervisor node
+    $ for i in `seq 1 6`; do vncviewer localhost:590$i & done
```

Comment:
Maybe specify that a ssh session needs to be running for the LocalForward lines to apply?